tests/test_syslog.py

```diff
@@ -0,0 +1,33 @@
+def test_parse_syslog():
+    from pgtoolkit.log.syslog import SyslogParser
+
+    lines = """\
+Mar 13 09:28:26 socle-dev0 postgresql-12-main[22163]: [1] [22163]: [1-1] db=,user=,app=,client= LOG:  automatic vacuum of table "postgres.pg_catalog.pg_default_acl": index scans: 1
+Mar 13 09:28:26 socle-dev0 postgresql-12-main[22163]: [2] #011pages: 0 removed, 2 remain, 0 skipped due to pins, 0 skipped frozen
+Mar 13 09:28:26 socle-dev0 postgresql-12-main[22163]: [3] #011tuples: 120 removed, 39 remain, 0 are dead but not yet removable, oldest xmin: 843
+""".splitlines(True)  # noqa
+
+    parser = SyslogParser(syslog_sequence_numbers=False)
+    parsed = list(parser.parse(lines))
+    assert len(lines) == len(parsed)
+    assert parsed[1].startswith('\t')
```

Comment:
might be worth some more assertions about attributes of `SyslogLine` items in `parsed`